### PR TITLE
closes #447 - missing link in notification emails

### LINKED ISSUE #447: missing link in notification emails
I got the following notification email. It leaves me unable to follow up with the comment, because there is no link

image

### PR DESCRIPTION
**Pull request by Knorrke**

### CODE REVIEW

File: src/module/Discussion/src/Discussion/Form/DiscussionForm.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ function __construct(HydratorPluginAwareDoctrineObject $hydrator, ObjectManager
             ]
         );
         $this->add(new Hidden('terms'));
-
+        $this->add(new Hidden('title'));
```
Comment (arekkas) [inline src/module/Discussion/src/Discussion/Form/DiscussionForm.php]: what is this for?

File: src/module/Ui/templates/discussion/partials/form-renderer.twig [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+        <div class="discussion-content-wrapper">
+            {{ formRow(element) }}
+        </div>
+    {% else %}
+            {{ formRow(element) }}
+    {% endif %}
+{% endfor %}
+{% for fieldset in form.getFieldsets() %}
+    {{ formCollection(fieldset) }}
+{% endfor %}
```
Comment (Knorrke) [inline src/module/Ui/templates/discussion/partials/form-renderer.twig]: should this be included? It adds the hidden fields of the discussion form and i think it makes things easier to change the form in the future... However this isn't needed to solve the issue.

Comment (arekkas) [inline src/module/Ui/templates/discussion/partials/form-renderer.twig]: hm, I think we don't really need it right now unless it is critical to get things working

**Comment by Knorrke** [comment]
i've removed the unnecessary changes @arekkas

**Pull request merged**